felix/dataplane/linux/bwp/bandwidth.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 CNI authors
```

Comment:
I like to put in a note "This code derived from the CNI bandwidth plugin, <link to repo>"

Lets us track it down later!